dist/shp.js

```diff
@@ -277,17 +277,21 @@ ParseShp.prototype.parseHeader = function() {
     ]
   };
 };
-ParseShp.prototype.getRows = function() {
+ParseShp.prototype.getRows = function(row) {
+  var rowFunc = row || function(data) { return data; };
   var offset = 100;
   var len = this.buffer.byteLength;
   var out = [];
   var current;
+  var transformedCurrent;
```

Comment:
this isn't used, I think you forgot to use it in line 292 and 293

Reply:
Thanks, updated now.